src/Platform/Microsoft.Testing.Platform/ServerMode/SerializerUtilities.cs

```diff
@@ -515,10 +516,11 @@ static SerializerUtilities()
 
         Deserializers[typeof(InitializeResponseArgs)] = new ObjectDeserializer<InitializeResponseArgs>(properties =>
         {
+            int processId = GetRequiredPropertyFromJson<int>(properties, JsonRpcStrings.ProcessId);
```

Comment:
is it a required property? or should we handle the case that it's missing or null?

Reply:
We force that property in the implementation so we have it always.